File: backend/src/main/java/com/bv87/diemdanh/dto/ScanLogItemDto.java
package com.bv87.diemdanh.dto;

import lombok.Builder;
import lombok.Getter;

import java.time.Instant;

/** One fingerprint scan log row for Admin/HEAD audit modal (SPEC §10.3). */
@Getter
@Builder
public class ScanLogItemDto {
    private final Instant scannedAt;
    private final String direction;
    private final Integer score;
    private final String message;
    private final String clientHostname;
    private final String clientIp;
    private final String kioskLabel;
}

File: desktop/src/BV87.Core/Models/Attendance/ScanLogItemDto.cs
namespace BV87.Core.Models.Attendance;

/// <summary>One fingerprint scan log row for Admin/HEAD audit modal.</summary>
public sealed class ScanLogItemDto
{
    public DateTimeOffset? ScannedAt { get; set; }
    public string? Direction { get; set; }
    public int? Score { get; set; }
    public string? Message { get; set; }
    public string? ClientHostname { get; set; }
    public string? ClientIp { get; set; }
    public string? KioskLabel { get; set; }
}
